src/ProjectService.Business/Commands/Task/GetTaskCommand.cs

```diff
@@ -40,15 +44,30 @@ public class GetTaskCommand : IGetTaskCommand
     private readonly ILogger<GetTaskCommand> _logger;
     private readonly IRequestClient<IGetCompanyEmployeesRequest> _rcGetCompanyEmployee;
     private readonly IRequestClient<IGetUsersDataRequest> _usersDataRequestClient;
+    private readonly IConnectionMultiplexer _cache;
     private readonly IRequestClient<IGetImagesRequest> _rcImages;
     private readonly IImageInfoMapper _imageMapper;
```

Comment:
remove _rcImages and _imageMapper

Reply:
??? why? we use it here

Reply:
sorry

Reply:
sorry. but in not instanticated -  you removed
_rcImages = rcImages;
_imageMapper = imageMapper;
see warnings.

Reply:
fixed